frontend/src/App.js

```diff
@@ -84,91 +104,43 @@ function App() {
       });
   };
 
-  sessionStorage.setItem(
-    "authToken",
-    new Buffer("Manager:Manager").toString("base64")
-  );
-
-  useEffect(() => {
-    logInUser();
-  }, [backendDomain]);
-
-  const logInUser = async () => {
-    //  todo: implement actual log in to get requests.
-    // todo: update this to not run every refresh.
-    // Getting hardcoded user from backend.
-    try {
-      console.log("Logging in");
-      const userPk = 3;
-      console.log("current domain: " + backendDomain);
-      console.log("Sending REquest: " + backendDomain + "user/" + userPk);
-      let userInfoResponse = await axios.get(backendDomain + "user/" + userPk, {
-        headers: {
-          Authorization: `basic ${sessionStorage.getItem("authToken")}`,
-        },
-      });
-      sessionStorage.setItem("user", JSON.stringify(userInfoResponse.data));
-    } catch (error) {
-      console.log(error);
-      window.alert("Could not sign in.");
-    }
-  };
-
   // TODO: Move this to a separate file
   const temp = [
```

Comment:
This variable could be moved to its own file and imported into App.js; it would help make the code look cleaner. You could take a look at what I did with CustomerSidebarItems.js for reference.

Reply:
Moved this into a function in new file in commit aac6818